shell/platform/linux/fl_renderer.cc

```diff
@@ -443,13 +448,21 @@ void fl_renderer_setup(FlRenderer* self) {
   }
 }
 
-void fl_renderer_render(FlRenderer* self, int width, int height) {
+void fl_renderer_render(FlRenderer* self,
+                        int width,
+                        int height,
+                        GdkRGBA* background_color) {
```

Comment:
Would it make sense to pass by const reference and require the caller provides a value? In other words, the view would be responsible for making black the default background color. I don't feel strongly about this, feel free to skip.

Reply:
Since this is C code, we should probably avoid references.

Reply:
Oh interesting. Noob question: why do we avoid references in C code? 👀

Reply:
References are a feature of C++ and aren't part of the C standard. Technically we could use them for functions that have internal linkage, but best to avoid putting them in any C APIs.

Reply:
Yes, it should be const (I did check this was the pattern for GdkRGBA, but forgot to set it here). It does seem a little cleaner to have the view set the default, I'll change that.